### PR TITLE
Use Case 5 

### PR DESCRIPTION
**Pull request by RocketSockz**

### CODE REVIEW

**Review by CernickyR — DISMISSED** [review]
Looks g2g.

File: Requirements for SSE.md [hunk 1]
```diff
@@ -98,22 +98,21 @@ Risks of installing third-party add-ons is difficult for Home Assistant to contr
...
 
-#### TODO: Misuse case Name
+#### Tablet Interface Use Case
 
-TODO: Insert info here
+Kaleb is one of Charlie's nephews whos more recently got into cyrbersecurity and hacking. Kaleb was extremely interested in Uncle Charlies integration with Home Assistant and while the adults where talking in the Kitchen, took Charlies Home Assistant integrated tablet to investigate. From the dashboard, Kaleb has access to the same configuration options as Charlie, more importantly access to all of the in network IoT devices, as well as their configurations. Kaleb also has access to the raw YAML configuration of Charlies tablet UI, allowing him to accidentally wipe out all of the customization Charlie has just set up. Kaleb also looks to add his own User to the Home Assistant System, which he can do, allowing him access to all devices in the system from his own phone whenever he is connected to Charlies local network.
```
Comment (Chrs987) [inline Requirements for SSE.md]: The paragraph is good just run through a spell checker

File: Requirements for SSE.md [hunk 2]
```diff
@@ -98,22 +98,21 @@ Risks of installing third-party add-ons is difficult for Home Assistant to contr
...
...nfiguration to be set up for tablet systems; However guest access is still a Work In Progress from Home Assistant, only allowing a yes or no to whether or not a user created is a UI administrator. In order to prevent other administrator actions, community created steps have been created to allow for a true guest user experience using Home Assistant, however it is important to note that this is NOT designed by Home Assistant. While this prevents Kalebs initial attack, arguements could be made that Charlie should have been notified to system changes to his Home Assistant, especially when his owner account credentials were use. Home Assistant does offer notifications from events happening in there [System Logs](https://www.home-assistant.io/integrations/system_log/), however those are only triggered with error or warning events NOT valid login or configuration changes. Eventually I wish for Home Assitant to provide security practicies similar to [CIS Benchamrks for AWS #1.1](https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cis-controls.html), that recommend the 'owner' account is never used, so that you can set up proper IAM access to systems through created Users.
```
Comment (Chrs987) [inline Requirements for SSE.md]: Home Assistant offers different users to be created to the system, allowing for a read-only configuration to be set up for tablet systems; - sentence is a little wonky reword it
 Eventually I wish for Home Assitant - put this in as HomeAssistant SHOULD not I wish just so it sounds better
Other than that just run it through a spell checker and its g2g

Comment (Chrs987) [reply Requirements for SSE.md]: The Admin and User accounts offered are done by HA not community. User accounts do not have access to edit addons or other configuration files, they are only able to access the User Interface and not change anything

**Comment by RocketSockz** [comment]
> Looks g2g.

I updated some stuff, mind giving it another look?

File: Requirements for SSE.md [hunk 3]
```diff
@@ -98,22 +98,21 @@ Risks of installing third-party add-ons is difficult for Home Assistant to contr
 
 ### Use Case 5
 
-#### TODO: Insert Use Case Name
+#### Tablet Interface Use Case
 
-TODO: Insert use-case info
+Charlie wants to be able to control his various IoT devices thorugh Home Assistant in the living room using a spare tablet. Home Assistant offers a customizable dashboards built in called [LoveLace](https://www.home-assistant.io/lovelace/), which allows for Charlie to set up a simple dashboard for his tablet, to control his various IoT devices with ease, such as his Amazon Alexa, Thermostat, and lights. Once installed Charlie is prompted for his login information to his account, Charlie being the only person set up in his Home Assistant, provides the login information that was orginially created during Home Assistants initaliazation, the Owner Account. Later that night Charlie invites his sister and her children over dinner, excited about his recent endevour, he shows his guests just how much control Home Assistant allows from his tablet.
```
Comment (Chrs987) [inline Requirements for SSE.md]: Home Assistant offers different users to be created to the system, allowing for a read-only configuration
this sentence is weirdly worded try:
"Home Assistant gives administrator users the ability to create standard user accounts..."
also 
remove I wish and say "Home Assistant should...."

**Review by CernickyR — COMMENTED** [review]
Reword "  Eventually I wish for Home Assistant to provide security practices similar to" to something more like " Home Assistant should ...... "

File: Requirements for SSE.md [hunk 4]
```diff
@@ -98,22 +98,21 @@ Risks of installing third-party add-ons is difficult for Home Assistant to contr
...
+To prevent Kaleb from doing all of these changes to Charlies Home Assistant integration, Home Assistant offers different users to be created to the system, allowing for a read-only configuration to be set up for tablet systems. While this prevents Kalebs initial attack, arguements could be made that Charlie should have been notified to system changes to his Home Assistant, especially when his owner account credentials were use. Home Assistant does offer notifications from events happening in there [System Logs](https://www.home-assistant.io/integrations/system_log/), however those are only triggered with error or warning events NOT valid login or configuration changes. Eventually I wish for Home Assitant to provide security practicies similar to [CIS Benchamrks for AWS #1.1](https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cis-controls.html), that recommend the 'owner' account is never used, so that you can set up proper IAM access to systems through created Users.
 
 #### Diagram
 
-TODO: Insert Diagram photo
-
+![Tablet Interface Use Case / Misuse Case Diagram](/images/Use-Case-5.png)
```
Comment (Chrs987) [inline Requirements for SSE.md:113]: change logs into to "Creates Guest Account" and you will be good

**Review by Chrs987 — DISMISSED** [review]
Once comments are fixed you are good to go

File: Requirements for SSE.md (hunk 2 above)
Comment (RocketSockz) [reply Requirements for SSE.md]: The documentation for Guest User or just Non-Administrative user creation is rather sparse, thanks for the clarification. I have updated this in the document.

File: Requirements for SSE.md (hunk 1 above)
Comment (RocketSockz) [reply Requirements for SSE.md]: Who knew that en devour was two separate words. I fixed all of the spelling mistakes

File: Requirements for SSE.md (hunk 3 above)
Comment (RocketSockz) [reply Requirements for SSE.md]: That has been changed!

**Review by Chrs987 — APPROVED** [review]
Very good. Thanks for changing the stuff

**Review by CernickyR — APPROVED** [review]
Reword under PREVENTION/SECURITY REQUIREMETNS  "Home Assistant gives administrator users the ability" to maybe "Home Assistant gives administrators the ability" - pretty much cutting out users as it seems kind of redundant.  A change that may be in the future but not necessary needed right now. -RC

**Pull request merged**